src/main/java/uk/gov/hmcts/reform/sandl/snlevents/actions/hearingpart/AssignHearingPartToSessionAction.java

```diff
@@ -155,4 +158,20 @@ private UserTransactionData getLockedSessionTransactionData(UUID id) {
         return new UserTransactionData("session", id, null, "lock", "unlock", 0);
     }
 
+    @Override
+    public List<ActivityLog> getActivities() {
+        List activities = new ArrayList();
```

Comment:
hym, this code looks very similar in every action. Can we make some utils from it?

Reply:
Discussed.